api/mock_trace_service.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package api;
+
+import "google/protobuf/empty.proto";
+
+option go_package = "github.com/googleinterns/cloud-operations-api-mock/api;mocktrace";
+
```

Comment:
Add this comment:
```// Next available id: 2```

imagine that there used to be a second field that has since been deleted -- there's compatibility issues if we ever reuse an id.

Reply:
The way the Go exporter works for testing is it instantiates the exporter and passes in the client connection info (see [here](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/8c1bbc5eb74bd29b602d739f88e98b185a13fc76/exporter/trace/cloudtrace_test.go#L90)). Right now after we spin up the mock server/clients with the library, there's no way to fetch the client info to pass to the exporter, so I added a getter to fetch this info.

Reply:
Done